screens/WaitingPage.js

```diff
@@ -42,6 +43,11 @@ export class WaitingScreen extends React.Component {
             text={ 'Delete\nRequest' }
             navigation={ this.props.navigation }
             destination={ 'Create' }
+            // onPress={() =>
```

Comment:
This commented out section is on the right track. The game should be deleted and then moved to the 'Selection' page. Why is this commented out?